tools/accuracy_checker/configs/person-reidentification-retail-0103.yml

```diff
@@ -0,0 +1,44 @@
+models:
+  - name: person-reidentification-retail-0103
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/person-reidentification-retail-0103/FP32/person-reidentification-retail-0103.xml
+        weights: intel/person-reidentification-retail-0103/FP32/person-reidentification-retail-0103.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   intel/person-reidentification-retail-0103/FP16/person-reidentification-retail-0103.xml
+        weights: intel/person-reidentification-retail-0103/FP16/person-reidentification-retail-0103.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - INT8
+        model:   intel/person-reidentification-retail-0103/INT8/person-reidentification-retail-0103.xml
+        weights: intel/person-reidentification-retail-0103/INT8/person-reidentification-retail-0103.bin
+        adapter: reid
+
+    datasets:
+      - name: globalme-reid
+        reader: pillow_imread
+
+        preprocessing:
+          - type: rgb_to_bgr
```

Comment:
Since we established in #633 that the models have BGR inputs after all, this should no longer be needed.

Reply:
Fixed.